eo-runtime/src/test/eo/org/eolang/string-tests.eo

```diff
@@ -140,3 +141,39 @@
   eq. > @
     "hello world".as-hash
     "hello world".as-hash
+
```

Comment:
@Graur try this pattern too: `/[0-9]+\/\d+/`. should match `15/3` for example.

Reply:
@yegor256 this pattern works only like this:
```
/([0-9]+/\\d+)/
.matches "15/3"
```
and my object `as-regex` works like this too:
```
"/([0-9]+/\\d+)/".as-regex
.matches "15/3"
```
About parenthesis I wrote [here](https://github.com/objectionary/eo/issues/675).